src/ui/components/pages/importDebug.tsx

```diff
@@ -0,0 +1,130 @@
+import { validators } from '@waves/waves-transactions';
+import { useAppDispatch, useAppSelector } from 'ui/store';
+import * as React from 'react';
+import { createAccount } from 'ui/actions';
+import { WalletTypes } from 'ui/services/Background';
+import { Button, Error, Input } from 'ui/components/ui';
+import * as styles from 'ui/components/pages/importDebug.module.css';
+import { useTranslation } from 'react-i18next';
+
+export function ImportDebug() {
+  const { t } = useTranslation();
+  const dispatch = useAppDispatch();
+  const accounts = useAppSelector(state => state.accounts);
+  const currentNetwork = useAppSelector(state => state.currentNetwork);
+  const customCodes = useAppSelector(state => state.customCodes);
+  const networks = useAppSelector(state => state.networks);
+  const networkCode =
+    customCodes[currentNetwork] ||
+    networks.find(n => currentNetwork === n.name).code;
+
+  const [showErrors, setShowErrors] = React.useState<boolean>(false);
+
+  const [name, setName] = React.useState<string>();
+  const [nameErr, setNameErr] = React.useState<string>();
+  const [address, setAddress] = React.useState<string>();
+  const [addressErr, setAddressErr] = React.useState<string>();
+
+  React.useEffect(() => {
+    setNameErr(t('importDebug.requiredError'));
+
+    if (name != null) {
```

Comment:
Will it turn to `null` if you fill it in and then erase? Why not check for empty string?